docs/merged_objects.md

```diff
@@ -46,6 +46,186 @@ rownames(merged_sce) # matrix row names
 
 There is also a `spliced` assay which contains the counts matrix with only reads from spliced cDNA.
 
+### SingleCellExperiment cell metrics
+
+
+Cell metrics calculated from the RNA-seq expression data are stored as a `DataFrame` in the `colData` slot, where row names are the cell barcode prefixed with the originating library id, e.g. `SCPCL000000-{barcode}`.
+This `DataFrame` also contains additional sample metadata information stored in the `colData` slot `DataFrame`, as described in [`Sample metadata` section](#singlecellexperiment-sample-metadata), for non-multiplexed samples.
```

Comment:
```suggestion
This `DataFrame` also contains additional sample metadata information stored in the `colData` slot `DataFrame` for all projects that do not contain multiplexed libraries.
Read more about the sample metadata that is included in the [`Sample metadata` section](#singlecellexperiment-sample-metadata), 
```
Since the presence of sample metadata is dependent on any libraries in a project being multiplexed, let's specify that.